.github/actions/labels/src/index.js

```diff
@@ -0,0 +1,44 @@
+const core = require("@actions/core");
+const github = require("@actions/github");
+const got = require("got");
+const process = require("process");
+
+const hyperjump_url = "http://github.aws.hlw3truzy4ls.com:6080/hyperjump/jump";
```

Comment:
My only question is: does this technically allow anyone to use these actions and piggyback on top of the service running at "http://github.aws.hlw3truzy4ls.com:6080/hyperjump/jump" for their own projects?

Reply:
Anyone can hit the hyperjump/jump url and request a jump to an arbitrary repo. The repository dispatch api request will fail unless the GitHub app is installed and.authorized for that repo. So repos have to opt-in to hyperjumps. However, any opted-in repo can receive hyperjumps from anywhere, so we limit the actions a hyperjump can do to actions any public user could normally perform, or that we don't mind them performing (like commenting and labeling PRs).